Source/WebCore/xml/parser/XMLDocumentParser.h

```diff
@@ -66,9 +66,9 @@ class XMLDocumentParser final : public ScriptableDocumentParser, public PendingS
     WTF_MAKE_FAST_ALLOCATED;
     WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR(XMLDocumentParser);
 public:
-    static Ref<XMLDocumentParser> create(Document& document, LocalFrameView* view, OptionSet<ParserContentPolicy> policy = DefaultParserContentPolicy)
+    static Ref<XMLDocumentParser> create(Document& document, bool isInFrameView, OptionSet<ParserContentPolicy> policy = DefaultParserContentPolicy)
```

Comment:
This goes against our usual style. We should not be passing boolean parameters like this. Either we keep the pointer or we switch to an enum class.

Reply:
I went for the enum option.